src/frontend/app/shared/data-services/cf-org-space-service.service.ts

```diff
@@ -170,6 +170,7 @@ export class CfOrgSpaceDataService {
   }
 
   public deleteSpace(spaceGuid: string, endpointGuid: string) {
-    return this.store.dispatch(new DeleteSpace(spaceGuid, endpointGuid));
+    this.store.dispatch(new DeleteSpace(spaceGuid, endpointGuid));
+    return;
```

Comment:
To nuke

Reply:
This function is used by the `cloud-foundry-space-base.component.ts`